python/src/aiconfig/default_parsers/gemini.py

```diff
@@ -329,6 +330,10 @@ async def run_inference(
 
         if not self.api_key:
             self.api_key = get_api_key_from_environment("GOOGLE_API_KEY", required=True).unwrap()
+
+        # Gemini api stores a reference to the currently executing async event loop.
+        # Reinitialize the "client" to avoid issues with the event loop.
+        self.model = genai.GenerativeModel(self.model_id)
```

Comment:
would prefer to have this under a "if TEMP_FIX_FOR_GEMINI_EVENT_LOOP_ISSUE:" and make "TEMP_FIX_FOR_GEMINI_EVENT_LOOP_ISSUE" true so people can easily just set this to false to repro and help us fix from Gemini side. Also pls submit a bug report to them and link it in code so people can follow up